user-dashboard.css

```diff
@@ -1,189 +1,235 @@
-@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800;900&display=swap');
+/* @import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800;900&display=swap"); */
 
 body {
-    background-color: rgba(169, 175, 197, 0.05);
-    font-family: Nunito;
+  background-color: rgba(169, 175, 197, 0.05);
+  font-family: Nunito;
 }
 
 .bg-blue {
-    background-color: #1F59BB;
+  background-color: #1f59bb;
+}
+
+.nav-link {
+  padding: 0;
 }
 
 .dashboard-wrap {
-    display: flex;
-    justify-content: space-between;
+  display: flex;
+  justify-content: space-between;
 }
 
 .dashboard-wrap > nav {
-    background-color: #1F59BB;
-    text-align: center;
-    max-width: 138px;
+  background-color: #1f59bb;
+  text-align: center;
+  max-width: 115px;
+  padding: 0;
 }
 
 .dashboard-content {
-    margin: 0 auto;
+  margin: 0 auto;
 }
 
-.dashboard-wrap  nav div {
-    text-align: center;
+.dashboard-wrap nav div {
+  text-align: center;
 }
 
 .top-bar {
-    display: flex;
-    align-items: center;
-    justify-content: flex-end;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
 }
 
 .top-bar nav .input-group .input-group-append > button:hover {
-    background: #1F59BB;
-    opacity: .9;
+  background: #1f59bb;
+  opacity: 0.9;
 }
 
 .top-bar .display-profile {
-    display: flex;
+  display: flex;
 }
 
 .top-bar .display-picture {
-    border-radius: 200px;
-    width: 50px;
-    height: 50px;
-    background: #fff;
+  border-radius: 200px;
+  width: 50px;
+  height: 50px;
+  background: #fff;
 }
 
-
 .dashboard-content .profile-summary {
-    background-image: url('img/svgs/profile-summary-bg.svg');
-    background-size: 100%;
-    background-repeat: no-repeat;
-    min-height: 20%;
-    background-position: 50% 50%;
-    margin-top: 20px; 
-    position: relative;
+  background-image: url("img/svgs/profile-summary-bg.svg");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  min-height: 20%;
+  background-position: 50% 50%;
+  margin-top: 20px;
+  position: relative;
 }
 
 .dashboard-content .profile-summary > p {
-    color: #fff;
-    position: absolute;
-    top: 36px;
-    left: 60px;
-    width: 90%;
-    min-height: 100%;
+  color: #fff;
+  position: absolute;
+  top: 36px;
+  left: 60px;
+  width: 90%;
+  min-height: 100%;
+}
+
+.main-top {
+  margin-bottom: 2rem;
+  padding: 0.75rem;
+  background: #5e81f4;
+  color: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5693342)